bench/gen_synthetic_dune_watch.ml

```diff
@@ -68,17 +68,21 @@ let write_lib ~base_dir ~lib ~dune =
 
 let write base_dir =
   let () = Unix.mkdir base_dir 0o777 in
-  let dune = {|
+  let dune =
```

Comment:
This seems a bit wasteful.

Reply:
This was changed in https://github.com/ocaml-ppx/ocamlformat/pull/2480 but perhaps with a different case in mind.
@dvulakh Do you have an opinion ?

Reply:
When formatting with `--profile=janestreet`, we want to keep the new behavior (multiline delimited strings always get their own lines).

We don't mind if `--profile=ocamlformat` or `--profile=default` keep some multiline strings docked, though.